helperFunctions/SalesforceMessageHelpers.ts

```diff
@@ -7,12 +7,34 @@ import { ErrorLogs } from '../enum/ErrorLogs';
 import { getAppSettingValue } from '../lib/Settings';
 import { agentTypingListener, removeAgentTypingListener } from './AgentTypingHelper';
 import { sendLCMessage } from './LivechatMessageHelpers';
+import { retrievePersistentTokens } from './PersistenceHelpers';
 
-export async function messageFilter(app: IApp, modify: IModify, read: IRead, messageRoom: IRoom, LcAgent: IUser, messageArray: any) {
+export async function messageFilter(app: IApp, modify: IModify, read: IRead, messageRoom: IRoom, LcAgent: IUser, messageArray: any, assoc, persistence) {
```

Comment:
Can you please also add type here `persistence: IPersistence`

`import { IHttp, IModify, IPersistence, IRead } from '@rocket.chat/apps-engine/definition/accessors';`

Also, can you move `persistence` after `read`? We should keep all the params(accessors/modifiers) from `app-engine` together.